src/biokbase/narrative/jobs/jobmanager.py

```diff
@@ -321,8 +321,7 @@ def _construct_job_output_state_set(
 
     def get_job_states(self, job_ids: List[str], ts: int = None) -> dict:
         """
-        Retrieves the job states for the supplied job_ids, with the option to
-        replace any jobs that have not been updated since ts with a short stub
```

Comment:
this documentation is for part of the method that hasn't yet been implemented => remove for now